docs/install/windows.md

```diff
@@ -1,37 +1,405 @@
+### Step 1: (If installing NEURON on a Windows Virtual Machine) 
+
+Create a new VM 
+  
+------------------------------------------------------------ 
+
+### Step 2: Install Git 
+
+Git: [https://git-scm.com/](https://git-scm.com/)
+
+Use this: [https://git-scm.com/download/win](https://git-scm.com/download/win)
+
+Note: Do not use: `winget install --id Git.Git -e --source winget` 
+ 
+  
+------------------------------------------------------------ 
+
+### Step 3: Install Chocolatey [https://chocolatey.org/](https://chocolatey.org/) by command line: 
+
+
+```powershell
+Set-ExecutionPolicy Bypass -Scope Process -Force; [System.Net.ServicePointManager]::SecurityProtocol = [System.Net.ServicePointManager]::SecurityProtocol -bor 3072; iex ((New-Object System.Net.WebClient).DownloadString('https://community.chocolatey.org/install.ps1')) 
+```
+This should be run in PowerShell or pwsh; in particular, it will not work in cmd.
+
+Ensuring Chocolatey commands are on the path 
+
+Ensuring chocolatey.nupkg is in the lib folder 
+ 
+ 
+------------------------------------------------------------ 
+
+### Step 4: Install Visual Studio with C++ 
+
+Use this: [https://visualstudio.microsoft.com/vs/features/cplusplus/](https://visualstudio.microsoft.com/vs/features/cplusplus/) 
+
+ 
+------------------------------------------------------------ 
+
+### Step 5: Get PowerShell 7 or newer version
+
+We will use `pwsh` command - correspond to PowerShell 7. The PowerShell 5 use `powershell` command.  
+ 
+#### Step 5.A: See PowerShell Version Currently Running: 
+
+This must be run inside whichever PowerShell environment you want to test.
+
+`$PSVersionTable` or `get-host|Select-Object`  
+ 
+#### Step 5.B: See available options: 
+
+`winget search Microsoft.PowerShell` 
+ 
+#### Step 5.C: Install New PowerShell 
+
+`winget install --id Microsoft.Powershell --source winget`  
+or
+`winget install --id Microsoft.Powershell.Preview --source winget` 
+ 
+#### Step 5.D: Ensure you are working with PowerShell 7
+
+Note: Find path file to version 7 (`C:\Program Files\PowerShell\7` or `C:\Program Files\PowerShell\7-preview`) and create a shortcut for this PowerShell 7
+Windows will continue to use 5 version if we try open "PowerShell" 
+
+PS: A shortcut is one option. You could also add it to your path. 
+
+------------------------------------------------------------ 
 
 ## Build Environment installation
 
-You can follow the lines of the scripts from the `ci` folder. The scripts can also be launched as-is for local installation. Perform a git clone of the `nrn` repository.
+### Step 6: Git clone
+At PowerShell 7, go to 
+`C:\Users\User>` and create folder named “Neuron”: 
+
+```
+mkdir Neuron
+```
+
+Inside the "Neuron” folder, do: 
+
+```
+git clone https://github.com/neuronsimulator/nrn
+```  
+
+ 
+Note: `PS C:\Users\User\Neuron> git clone git@github.com:neuronsimulator/nrn nrn` may be an option
```

Comment:
This only works if you have shared a public key with GitHub. The previous version should always work. Generally, I'd suggest focusing on the simpler, shorter way that always works and avoiding things that  require other assumptions to keep the how-to guide clear,